dvc/scm/backend/gitpython.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from typing import Optional, Tuple
+
+from dvc.scm.backend.base import BaseGitBackend
+
+logger = logging.getLogger(__name__)
+
+
+class GitPythonBackend(BaseGitBackend):  # pylint:disable=abstract-method
+    """git-python Git backend."""
+
+    @property
+    def git(self):
+        return self.scm.repo.git
```

Comment:
This plus the empty `__init__` are placeholders until all of the current `dvc.scm.git` git-python functionality is moved into this backend (or removed entirely)